src/PokeUsers.class.js

```diff
@@ -0,0 +1,14 @@
+const PokeUser = require('./PokeUser.class')
```

Comment:
Prefer creating a collection class that can embed any class. And add a static on PokeUser class that use this collection to create many users.